vhost-device-scmi/src/devices/iio.rs

```diff
@@ -390,11 +605,11 @@ impl IIOSensor {
         // To get meaningful integer values, we must adjust exponent to
         // the provided scale if any.
         let custom_exponent = self.custom_exponent(path, unit_exponent);
-        axes.push(Axis {
-            path: OsString::from(path),
+        axes.push(Axis::new(
+            OsString::from(path),
             unit_exponent,
             custom_exponent,
-        });
+        ));
```

Comment:
Ditto, are this change related?

Reply:
No, this change is a new feature.
I implement the function "new" for "Axis", it can help us to get "scan_type" from sysfs for each axis, if this attribute exists. It is used for notification flow.

Reply:
No, this change is a new feature.
I implement the function "new" for "Axis", it can help us to get "scan_type" from sysfs for each axis, if this attribute exists.